reflections/2020-05-22.md

```diff
@@ -0,0 +1,44 @@
+# Reflections
+
+## Participation
+
+- 3 groups 6-8 people in size.
+- Each group had a brainhack committee member and a python/programming enthusiast/expert
+
+## What Happened
+
+- some groups were quiet from the onset and didn't have questions to open up discussion
+- one group had the brainhack member share their screen and run through the lesson
+- another group had questions about what editor they should use (spyder/jupyter notebooks)
+- some participants did not know they could edit their google colab notebooks
+  - they were afraid or unsure if they commented in their colab notebooks, that the
+    changes would affect everyone else? Additionally, they were unsure how to copy,
+    download, and choose the format from options provided for re-use later.
+- some participants are working ahead, and some are sticking to the schedule
+- There seemed to be questions on how to interpret different colors (something
+  specific to colab but not necessarily a feature only found in colab)
+  for say 'string' versus string. (colors can represent a data type)
+- There was also questions about how a variables value can be modified.
+  - For example: weight_kg = 100, what happens to weight_kg when I run --
+    weight_kg * 2.2 = weight_lb?
+
```

Comment:
I just remembered, running notebook cells out of order also came up, which is covered in this talk ([among the other less desired features of notebooks](https://www.youtube.com/watch?v=7jiPeIFXb6U))